docs/BreakingChanges/072 SqlClient collates sql_variant now.md

```diff
@@ -0,0 +1,22 @@
+## 72: SqlClient collates sql_variant now
```

Comment:
{
  "Id": "72",
  "Title": "SqlClient collates sql_variant now",
  "Details": "`sql_variant` data uses `sql_variant` collation rather than database collation.",
  "Suggestion": "This change addresses possible data corruption if the database collation differs from the `sql_variant` collation. Applications that rely on the corrupted data may experience failure.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx?f=255&MSPPError=-2147217396#xml",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": false,
  "ImpactScope": 4
}

**Parsed APIs:**
- [ ] Not detectable via API analysis